Kudu.Core/Helpers/ProjectGuidParser.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+
+/// <summary>
+/// This file is from https://github.com/vijayrkn-test/ProjectGuidParser/blob/dfa4bbdc0d163b46684e1438fb4f5cb972bfcb71/ProjectGuidParser/ProjectGuidParser.cs
+/// </summary>
+namespace Microsoft.VisualStudio.Web
```

Comment:
I think we should change the namespace to ours to make it clear that this code lives in Kudu.